spec/operators/filter-spec.ts

```diff
@@ -1,6 +1,12 @@
 import {expect} from 'chai';
 import * as Rx from '../../dist/cjs/Rx';
-declare const {hot, cold, asDiagram, expectObservable, expectSubscriptions};
+import marbleTestingSignature = require('../helpers/marble-testing'); // tslint:disable-line:no-require-imports
```

Comment:
could you not just destructure it with the import?

`import {hot, cold, asDiagram, expectObservable, expectSubscriptions} from '../helpers/marble-testing';`